scripts/nn/test/compare_backends/test_conv2d.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+export SPARK_HOME=~/spark-2.1.0-bin-hadoop2.7
```

Comment:
I think it would be good to remove this and just assume the user will have the Spark `bin` folder in the path.  That way, this will be able to be run by anyone, rather than forcing another dev to have this specific Spark directory available.  Same thing for the other shell scripts.